labonneboite/tests/web/api/test_api.py

```diff
@@ -993,7 +1059,56 @@ def test_no_labonneboite_url_with_coordinates(self):
             rv = self.app.get('%s?%s' % (url_for("api.company_list"), urlencode(params)))
             self.assertEqual(rv.status_code, 200)
             data = json.loads(rv.data)
-            self.assertEqual(data['url'], '')
+
+            # ensure we return home URL and not search URL
+            self.assertNotIn('/entreprises', data['url'])
+            self.assertIn('http://localhost:5000/?utm', data['url'])
```

Comment:
Ne faudra t-il pas utiliser url_for('home') ?

Reply:
ouais je vais faire mieux

Reply:
fixed in my last commit